packages/repeater/src/api/commands/ListRepeatersRequest.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpRequest } from '@secbox/bus';
+
+type ListRepeatersResponsePayload = { id: string; name: string }[];
```

Comment:
Perhaps, it should be a public type 
```suggestion
export type ListRepeatersResponsePayload = { id: string; name: string }[];
```